pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -61,6 +63,11 @@ const (
 	OSDOverPVCLabelKey                  = "ceph.rook.io/pvc"
 )
 
+var (
+	osdPodPrepareMemory = int64(display.MbTob(200))
+	osdPodPrepareCPU    = int64(1000)
```

Comment:
Is this `1000m`? Following best practice guidelines, the variable should indicate its units. e.g., `osdPodPrepareMemoryBytes` and `osdPodPrepareCPUMillicores`.

Reply:
From what I understood, the Millicores unit is a CPU core split into 1000 units (milli = 1000). So 1 CPU = 1000. At his point, I'm allocating 1 CPU core.

Also `NewMilliQuantity()` returns (1000*1/1000) so 1.